cmd/channel/email/main.go

```diff
@@ -80,7 +81,60 @@ func (ch *Email) SetConfig(jsonStr json.RawMessage) error {
 }
 
 func (ch *Email) GetInfo() *plugin.Info {
-	return &plugin.Info{Name: "Email"}
+	elements := []*plugin.ConfigOption{
+		{
+			Name: "host",
+			Type: "string",
+			Label: map[string]string{
+				"en_US": "SMTP Host",
+				"de_DE": "SMTP Host",
+			},
+		},
+		{
+			Name: "port",
+			Type: "option",
```

Comment:
Wouldn't it make more sense to have a freely selectable port, because otherwise someone will definitely be unhappy? What about the selection of the transport encryption? This cannot be read from the port per se.